src/env.rs

```diff
@@ -16,6 +16,8 @@ pub struct Config {
     pub tenant_database_url: Option<String>,
     #[serde(default = "default_tenant_id")]
     pub default_tenant_id: String,
+    #[serde(default = "default_is_test")]
+    pub is_test: bool,
```

Comment:
Please use serde ignore so that people cannot send this as an env var and hide it at the bottom of the config struct

Reply:
You mean `serde skip`? https://serde.rs/field-attrs.html#skip

Reply:
yah